apps/studio/tests/integration/helpers/seed/index.ts

```diff
@@ -0,0 +1,264 @@
+        codeBuildId: null,
+        theme: null,
+      })
+      .returningAll()
+      .executeTakeFirstOrThrow()
+
+    // Insert navbar
+    const navbar = await tx
+      .insertInto("Navbar")
+      .values({
+        content: jsonb([
+          {
+            url: "/item-one",
+            name: "Expandable nav item",
+            items: [
+              {
+                url: "/item-one/pa-network-one",
+                name: "PA's network one",
+                description:
+                  "Click here and brace yourself for mild disappointment.",
+              },
+              {
+                url: "/item-one/pa-network-two",
+                name: "PA's network two",
+                description:
+                  "Click here and brace yourself for mild disappointment.",
+              },
+              { url: "/item-one/pa-network-three", name: "PA's network three" },
+              {
+                url: "/item-one/pa-network-four",
+                name: "PA's network four",
+                description:
+                  "Click here and brace yourself for mild disappointment. This one has a pretty long one",
+              },
+              {
+                url: "/item-one/pa-network-five",
+                name: "PA's network five",
+                description:
+                  "Click here and brace yourself for mild disappointment. This one has a pretty long one",
+              },
+              {
+                url: "/item-one/pa-network-six",
+                name: "PA's network six",
+                description:
+                  "Click here and brace yourself for mild disappointment.",
+              },
+            ],
+          },
+        ]),
+        siteId: site.id,
+      })
+      .returningAll()
+      .executeTakeFirstOrThrow()
+
+    // // Insert footer
+    const footer = await tx
+      .insertInto("Footer")
+      .values({
+        content: jsonb({
+          siteNavItems: [
+            { url: "/about", title: "About us" },
+            { url: "/partners", title: "Our partners" },
+            { url: "/grants-and-programmes", title: "Grants and programmes" },
+            { url: "/contact-us", title: "Contact us" },
+            { url: "/something-else", title: "Something else" },
+            { url: "/resources", title: "Resources" },
+          ],
+          contactUsLink: "/contact-us",
+          termsOfUseLink: "/terms-of-use",
+          feedbackFormLink: "https://www.form.gov.sg",
+          privacyStatementLink: "/privacy",
+        }),
+        siteId: site.id,
+      })
+      .returningAll()
+      .executeTakeFirstOrThrow()
+
+    return { site, navbar, footer }
+  })
+}
+
+export const setupBlob = async (blobId?: string) => {
+  if (blobId !== undefined) {
+    return db
+      .selectFrom("Blob")
+      .where("id", "=", blobId)
+      .selectAll()
+      .executeTakeFirstOrThrow()
+  }
+  return db
+    .insertInto("Blob")
+    .values({
+      content: jsonb({
```

Comment:
would it make sense to extract all of the values into constants to keep these queries clean (similar to curr seed script)

Reply:
next time ba. i think not unwieldy yet and still readable. I favor readability over everything for tests